src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int size = 0;
+    private final int defaultSize = 10;
```

Comment:
make this field a constant (add static modifier). remember the naming for constants and ordering (constants should introduced before regular class fields.